### ISSUE TITLE
WDLCompanion doesn't work with WDL Bypass. BUG

### ISSUE THREAD

**Issue by facundogaidano** [Pokechu22/WorldDownloader-Serverside-Companion#5]
Hello, I found a WDL "bypass" that with this Bypass your plugin doesn't detect it and users can download the map.
I wanted to found ways how users can join to my server with WLD and this is the only way that they can join (by "WLD Undetectable/Cracked"). I hope you can fix it :)

**Comment by Pokechu22** [comment]
I'm aware of this issue, but there isn't anything I can do on the plugin side; I've been trying to handle it via legal means (redistribution of modified versions of WDL is in violation of the mod's license) but that's a slow process.

**Comment by facundogaidano** [comment]
So what can I do? I don't want my builders waste time If someone can join to the server and download their hard work.
 I'm working on an MC server and It's a ROL server with gigantic constructions. If users can download the map it's a big problem for the server :(

**Comment by Pokechu22** [comment]
Non-technical means.  Make it clear that it's your map, by including signs and other contact information in it.  And, you can send legal notice to those who are re-distributing it, since they would be violating your copyright.  Which is more of a problem, that someone has a local copy of your map (which you get when you join the server anyways), or that someone is using that map on another server?
